http_server/_test_connection_handling.pony

```diff
@@ -1,5 +1,5 @@
-use "pony_test"
```

Comment:
This is unrelated at this point, no new uses were added so I think this should be rolled back